src/index.html

```diff
@@ -22,6 +22,384 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo image"
+          class="header__logo-img"
+        />
+      </a>
+
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__picture"
+        />
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <h3 class="card__subtitle">Product code: 195434</h3>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
```

Comment:
Try to use ul and li in this case.
it's preferred because we have one repeated element inside div.
In this case best variant use list)